app/assets/stylesheets/content/_advanced_filters.sass

```diff
@@ -72,6 +72,11 @@
       @include form--input-field-mixin--narrow
       flex: 0 0 auto
 
+.advanced-filters--add-filter
+  display: grid
+  grid-template-columns: 20% 45% 50px
+  grid-gap: 10px
```

Comment:
We should start introducing variables here so that the gap and column distributions do not become magic numbers.

Reply:
BTW: Funktionality is good, bug is fixed. :)